apps/coa/behaviours/save-desired-content.js

```diff
@@ -0,0 +1,18 @@
+module.exports = superclass => class extends superclass {
+  locals(req, res) {
+    const locals = super.locals(req, res);
+    // nameWithPossession returns the name which puts an ('s) on the end of the applicant's name.
+    // If the name ends with S, its just returns the applicant's name with apostrophe (') at end.
+    const applicantName = req.sessionModel.get('applicant-full-name');
+    const nameWithPossession = applicantName + (applicantName.toLowerCase().endsWith('s') ? '’' : '’s');
+    req.sessionModel.set('nameWithPossession', nameWithPossession);
+    return locals;
```

Comment:
`return super.getValues(req, res, next);`

Reply:
changes made as per the comment